rsocket-routing-http-bridge/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,3 @@
+# AutoConfiguration
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+io/rsocket/routing/http/bridge/config/RSocketHttpBridgeAutoConfiguration
```

Comment:
`io.rsocket.routing.http.bridge.config.RSocketHttpBridgeAutoConfiguration` change slashes to dots